homeworks/homework-vue/.storybook/preview.js

```diff
@@ -0,0 +1,4 @@
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
```

Comment:
Used to automatically map events starting with `on` e.g. `onUpdate` to actions in storybook. Could be removed since not used